scripts/nebula-storaged-listener.service

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+
+SCRIPT_PATH=$(readlink -f $0)
```

Comment:
I think you'd better to check if SCRIPT_PATH is soft link in a while loop, for $(readlink -f $SCRIPT_PATH) may also be a soft link

Reply:
ok good